ember-learn/ember-blog
# The Ember Times No. 153 - June 19th 2020

📝  Ideas, feel free to add to list or claim!

- RFC roundup:
  - [x] Type Stability for Addons (🔒 @chrisrng )
    - https://github.com/typed-ember/ember-cli-typescript/pull/1158
- [x] Shibu Lijack (https://github.com/shibulijack) created a LinkedIn group that welcomes anyone who wants to share and learn the latest and greatest Ember.
  - https://www.linkedin.com/posts/shibulijack_ember-community-strongertogether-activity-6673898303190654976-BgRg (🔒 @abhilashlr )
  - Shared a link to [how to migrate away from mixins and identify memory leaks](https://shibulijack.gitbook.io/ember-best-practices/) (🔒 @abhilashlr )
- [x] `ember-container-query` released (:lock: @ijlee2)
  - https://github.com/ijlee2/ember-container-query/
  - Isaac wrote a few blog posts related to the addon. [Adaptive Images](https://crunchingnumbers.live/2020/06/03/container-queries-adaptive-images/) and [Cross-Resolution Testing](https://crunchingnumbers.live/2020/06/07/container-queries-cross-resolution-testing/) may be interesting to look at.
- [x] Becoming a Developer, a website built in Ember that encourages people to share how they started working in development (Reserved for :lock: @roderickdevelopers)
  - https://dev.to/agathebadia/share-your-dev-story-51dg
- [x] from @atej on Discord: Hi, I made a bash script to automate setting up a new ember project with various bells and whistles for a smooth DX inside VS Code on a Mac - https://github.com/atej/ember-new-plus (:lock: @atej)
  - It set's up eslint-config-airbnb, prettier, tailwindcss+postcss, husky, lint-staged, commitlint and debugger for chrome with VS Code integrations (wherever applicable) in a jiffy.
  - Everything is optional (via selection prompts) and the config files can be edited after the initial set-up.
  - This saves me a bunch of time. Hope other ember devs find it useful too!
- [x] Understanding Ember.js Public Assets Path for Browser and Cordova
  - https://dev.to/betocantu93/understanding-ember-js-public-assets-path-jj8 (:lock_with_ink_pen: @jayjayjpg)
- [ ] A new release of `ember-place-autocomplete`
  - https://twitter.com/dmuneras/status/1265530766739603457
- [ ] v1.2 release of `ember-concurrency` and v2.0 of `ember-concurrency-decorators`
  - https://twitter.com/m4xm4n/status/1273404988845367297
- [ ] New addon: `ember-validation-state`
  - https://github.com/chrismllr/ember-validation-state
  - > Just released an addon which provides a decorator for form validation, on top of ember-validators:
     >
     > Utilizes patterns defined in ember-cp-validations, and brings in some of the intl support a la ember-intl/cp-validations
- [x] Ember A11y Strike Team will have their last meeting next week. The Working Group for Digital Accessibility will form to help developers with a11y on a long-term basis. (:lock: @jaredgalanis )
  - https://github.com/ember-a11y/core-notes/blob/ember-a11y/ember-a11y/2020-06/june-17.md
  - Reach out to @MelSumner for more information. Let's find out who worked on the Strike Team so that we can thank them in the newsletter. 

📆  Future stories

- Also a heads up, the roadmap RFC was merged but a blog post is upcoming and that's what should be publicised (from @locks - follow this PR https://github.com/ember-learn/ember-blog/pull/646)
- Deprecate string based actions RFC https://github.com/emberjs/rfcs/pull/632, check on messaging with locks 
- Updates on Yehuda Octane series (when new posts after April 6 one are out): https://yehudakatz.com/2020/04/06/ember-octane-airtable-time/
- Updates on pzuraq autotracking series https://www.pzuraq.com/autotracking-case-study-trackedmap/
- Rock and Roll with Ember.js Octane edition will be released on 6/25
  - https://twitter.com/baaz/status/1273975698299006978

🤔  Questions?

See #support-ember-times on Discord or CONTRIBUTING.

✅  Todos:

- Check https://github.com/ember-learn/whats-new-in-emberland (roundup of all the PRs in emberjs and ember-learn repos) for ideas
- Use https://github.com/ember-learn/emberjs-times-tools README
- Check Netlify preview, to ensure display on emberjs.com/blog is good
  - [ ] Check buttons
- [ ] Crosspost on dev.to, delete Readers Question picture, add Canonical URL 
- [ ] Set up tweet for Monday morning on tweetdeck
  - [ ] post $natural
